src/main/java/Duke/command/AddCommand.java

```diff
@@ -0,0 +1,44 @@
+package Duke.command;
+
+import Duke.model.Deadline;
+import Duke.model.Event;
+import Duke.model.ToDo;
+import Duke.task.Task;
+import Duke.task.TaskList;
+import Duke.utils.Ui;
+
+public class AddCommand extends Command {
+    String taskType;
```

Comment:
Shall we privatize variables for encapsulation? If I'm not mistaken, its part of the coding standard that this mod enforces